content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -265,8 +267,7 @@ well as lower-trust users.The following listed controls should be enforced/disal
 				spec.containers[*].securityContext.seccompProfile<br>
```

Comment:
Note that the first line here says seccompProfile.type while the second one only says seccompProfile.

I believe ".type" needs to be added on both of these lines that lack it, but I recommend you play with it yourself in a test cluster to make sure you update it to what is both modern and correct.

Reply:
yes, the path is `...securityContext.seccompProfile.type`